docs/references/recipes.md

```diff
@@ -271,3 +271,50 @@ We made it very simple to debug your app with VS Code.
 5. Click the green "Start debugging" button located next to the dropdown
 
 6. That's it. VS Code should run your code and stop wherever your set some breakpoints
+
+## Use with Next.js
```

Comment:
To avoid error prone duplication of technical content I want to have the Next.JS Nexus examples link to this in their READMEs.